README.md

```diff
@@ -67,6 +67,29 @@ commands:
 
 Full installation instructions are available in [INSTALL.md](https://github.com/metabrainz/acousticbrainz-server/blob/master/INSTALL.md) file.
 
+## Login and Datasets
```

Comment:
We talked about this on IRC. Setup information should be in README. INSTALL is only for manual server install, then it should refer to readme for setup.